services/algolia-sync/src/env.js

```diff
@@ -13,4 +13,5 @@ module.exports = cleanEnv(process.env, {
   ALGOLIA_API_KEY: nonemptystr({ desc: 'Algolia API Key' }),
   AWS_ACCESS_KEY_ID: nonemptystr({ desc: 'AWS Access Key' }),
   AWS_SECRET_ACCESS_KEY: nonemptystr({ desc: 'AWS Secret' }),
+  AWS_AWS_REGION: nonemptystr({ desc: 'AWS Region', default: 'us-east-1' }),
```

Comment:
Lets make this just `AWS_REGION` to match the standard env var that the sdk & cli listen to

Reply:
oops didn't mean to do that.